paymentsheet/src/test/java/com/stripe/android/paymentsheet/ui/EditPaymentMethodUiScreenshotTest.kt

```diff
@@ -0,0 +1,92 @@
+package com.stripe.android.paymentsheet.ui
+
+import com.stripe.android.model.CardBrand
+import com.stripe.android.utils.screenshots.FontSize
+import com.stripe.android.utils.screenshots.PaparazziRule
+import com.stripe.android.utils.screenshots.PaymentSheetAppearance
+import com.stripe.android.utils.screenshots.SystemAppearance
+import org.junit.Rule
+import org.junit.Test
+
+class EditPaymentMethodUiScreenshotTest {
+    @get:Rule
+    val paparazziRule = PaparazziRule(
+        SystemAppearance.values(),
+        PaymentSheetAppearance.values(),
+        FontSize.values(),
```

Comment:
When testing an entire screen (as opposed to a single component), we typically limit tests to only a single configuration (e.g. light theme + default appearance + default font size). Can you make that change?